packages/trpc/server/routers/viewer/availability.tsx

```diff
@@ -9,259 +9,265 @@ import { Schedule } from "@calcom/types/schedule";
 
 import { TRPCError } from "@trpc/server";
 
-import { createProtectedRouter } from "../../createRouter";
+import { router, authedProcedure } from "../../trpc";
 
-export const availabilityRouter = createProtectedRouter()
-  .query("list", {
-    async resolve({ ctx }) {
-      const { prisma, user } = ctx;
+export const availabilityRouter = router({
+  list: authedProcedure.query(async ({ ctx }) => {
+    const { prisma, user } = ctx;
 
-      const schedules = await prisma.schedule.findMany({
-        where: {
-          userId: user.id,
-        },
-        select: {
-          id: true,
-          name: true,
-          availability: true,
-          timeZone: true,
-        },
-        orderBy: {
-          id: "asc",
-        },
-      });
+    const schedules = await prisma.schedule.findMany({
+      where: {
+        userId: user.id,
+      },
+      select: {
+        id: true,
+        name: true,
+        availability: true,
+        timeZone: true,
+      },
+      orderBy: {
+        id: "asc",
+      },
+    });
 
-      const defaultScheduleId = await getDefaultScheduleId(user.id, prisma);
+    const defaultScheduleId = await getDefaultScheduleId(user.id, prisma);
 
-      return {
-        schedules: schedules.map((schedule) => ({
-          ...schedule,
-          isDefault: schedule.id === defaultScheduleId,
-        })),
-      };
-    },
-  })
-  .query("schedule", {
```

Comment:
since we have a router called schedule, we cannot have a procedure named the same, so this procedure is now schedule.get